website/project/model.py

```diff
@@ -760,7 +757,7 @@ def __init__(self, reason, key, *args, **kwargs):
 
 def validate_doi(value):
     # DOI must start with 10 and have a slash in it - avoided getting too complicated
-    if not re.match('10\\.\\S*\\/', value):
+    if value and not re.match('10\\.\\S*\\/', value):
```

Comment:
That was relating to the regex being not-great. This was added to allow someone to unset `preprint_article_doi` -- the validator didn't allow `''`